api/channel.go

```diff
@@ -1172,6 +1172,10 @@ func updateNotifyProps(c *Context, w http.ResponseWriter, r *http.Request) {
 		member.NotifyProps["desktop"] = desktop
 	}
 
+	if push, exists := data["push"]; exists {
```

Comment:
Super minor but can we use `ok` instead of `exists`? We use `ok` everywhere else and it's kind of an unwritten convention in Go